wagtail/wagtailadmin/static_src/wagtailadmin/scss/layouts/404.scss

```diff
@@ -0,0 +1,101 @@
+@import 'wagtailadmin/scss/variables';
+@import 'wagtailadmin/scss/mixins';
+@import 'wagtailadmin/scss/grid';
+@import 'wagtailadmin/scss/components/logo';
+
+body {
+    font-family: $font-sans;
+    color: $color-white;
+
+    .page404__bg {
```

Comment:
There is no need for this to be nested within `body`. Please don't use Sass nesting.